src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -119,6 +119,15 @@ def _stopped_handler(self, events: List[CloudEvent]) -> None:
         if self.ensemble.status != ENSEMBLE_STATE_FAILED:
             self._result = events[0].data  # normal termination
             with self._snapshot_mutex:
+                max_memory_usage = -1
+                for job in self.ensemble.snapshot.get_all_jobs().values():
+                    memory_usage = job.max_memory_usage or "-1"
+                    if int(memory_usage) > max_memory_usage:
+                        max_memory_usage = int(memory_usage)
+                logger.info(
+                    "Experiment ran with maximum memory usage for a single realization job: %s",
```

Comment:
This is outputted for each ensemble, not the entire experiment (which can contain 1 or more ensembles)?

Reply:
Do you think we should log this on a ensemble basis, or move it up to the experiment?

Reply:
Ensemble is probably fine.